webui/react/src/components/kit/Icon.test.tsx

```diff
@@ -17,7 +17,9 @@ const svgIcons = [
 
 const setup = (props?: Props) => {
   const user = userEvent.setup();
-  const view = render(<Icon name="star" showTooltip title="Icon" {...props} />);
+  const view = render(
+    <Icon color={props?.color} name="star" showTooltip size={props?.size} title="Icon" />,
```

Comment:
pass in the name of the icon from props as well:
```suggestion
    <Icon color={props?.color} name={props?.name} showTooltip size={props?.size} title="Icon" />,
```